pkg/cloud/vsphere/context/ssh.go

```diff
@@ -0,0 +1,61 @@
+package context
```

Comment:
Hi @moshloop,

Please see my previous comment about SSH. Also, on new files please add the license header.